### ISSUE TITLE
Inconsistency in chebyshevcenter documentation

### ISSUE THREAD

**Issue by schillic** [JuliaPolyhedra/Polyhedra.jl#185]
`chebyshevcenter` either calls `hchebyshevcenter` or `vchebyshevcenter` depending on the argument. However, according to the documentation, these two functions compute different things.

https://github.com/JuliaPolyhedra/Polyhedra.jl/blob/00173cec89b318ec2d06b585e7b8510a91e130f6/src/center.jl#L5-L7

https://github.com/JuliaPolyhedra/Polyhedra.jl/blob/00173cec89b318ec2d06b585e7b8510a91e130f6/src/center.jl#L53-L55

According to [Wikipedia](https://en.wikipedia.org/wiki/Chebyshev_center), these two definitions are not equivalent in general.
> In geometry, the Chebyshev center of a bounded set Q having non-empty interior is the center of the minimal-radius ball enclosing the entire set Q, or alternatively (and non-equivalently) the center of largest inscribed ball of Q.

Is this intended?

**Comment by blegat** [comment]
No, V-chebyshev center should do something different but I never got the time to implement it.
